### PR TITLE
Using discovery 1.0

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
+        "php": "^5.5 | ^7.0",
```
Comment (sagikazarmark) [inline composer.json]: Double pipes please

**Comment by sagikazarmark** [comment]
Let's wait for the react adapter then merge this.

**Comment by Nyholm** [comment]
The react client is updated. Builds are passing

**Comment by sagikazarmark** [comment]
I know, but I want to update the meta files.

**Comment by Nyholm** [comment]
That is not a blocker for this PR. Let's merge this now.

**Pull request merged**